### DISCUSSION TITLE
Composer detected issues in your platform

### DISCUSSION THREAD

**Discussion by hamzaelmaghari** [laravel/framework#39518] [Q&A]
- Laravel Version: 8.65
- PHP Version: 8.0.12
- Database Driver & Version: MySQL 8.0.19

### Description:
![Screen Shot 2021-11-07 at 14 19 17](https://user-images.githubusercontent.com/84163115/140646981-b3a373af-4313-411d-a82a-4855cb33e834.png)
I just want to make sure to know if this problem comes from Laravel Framework, or Composer itself.
I was working on my project as usual, when I woke up this morning I started the SQL engine, Valet starts auto.
I got this error:
`Fatal error: Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.0". You are running 7.4.25. in /Users/user/Desktop/sites/marketplace/vendor/composer/platform_check.php on line 24
`
Is there anyone else faces this issue or just me?

**Comment by d3n972** [comment]
Hi, this error is from Composer, you should check which php are you using in your web server configuration, because it's looks like, you have 7.4.25 and 8.0.12 installed.

**Reply by hamzaelmaghari** [reply]
Thanks for your help, I'll try to remove 7.4.0 and keep just 8.
I temporarily fix this issue by running command:
`Valet use php --force`
